contracts/scripts/DeployScripts.s.sol

```diff
@@ -26,6 +26,7 @@ contract DeployScript is Script {
             0x68bC10674b265f266b4b1F079Fa06eF4045c3ab9
         );
         uint16 feePercent = 2;
+        uint16 slashPercent = 5;
```

Comment:
The above change will be n/a with https://github.com/primev/mev-commit/pull/293, but we can use this change